anker/src/state.rs

```diff
@@ -40,6 +41,9 @@ pub struct WormholeParameters {
     Clone, Debug, Default, BorshDeserialize, BorshSerialize, BorshSchema, Eq, PartialEq, Serialize,
 )]
 pub struct Anker {
+    /// Version number for Anker.
+    pub version: u8,
```

Comment:
I wondered where it got set, but then I realized it’s the `Default` impl. That’s a bit subtle, can we initialize it explicitly in `process_initialize`?

Reply:
Never mind, it is there, I missed it. LGTM then!